pkg/machine/e2e/basic_test.go

```diff
@@ -113,7 +113,7 @@ var _ = Describe("run basic podman commands", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(volumeCreate).To(Exit(0))
 
-		run, err := mb.setCmd(bm.withPodmanCommand([]string{"run", "-v", "a:/test:Z", "quay.io/libpod/alpine_nginx", "true"})).run()
+		run, err := mb.setCmd(bm.withPodmanCommand([]string{"run", "-v", "a:/test:Z", "quay.io/libpod/alpine", "true"})).run()
```

Comment:
please use TESTIMAGE instead of yet another image that is not well maintained

Reply:
Thank you. I have updated that (cc @axel7083)